### PR TITLE
Few minor optimizations and few coding convention fixes (+ 10.4 Sydney support)

### PR DESCRIPTION
**Pull request by mWaltari**
Could not (yet) figure out how to make these commits into few smaller pull requests, but hope this will do

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,3 +24,4 @@ lib
 /Documentation/graphics32.hhp
 Win32
 Win64
+*.ficfg
```
Comment (andersmelander) [inline .gitignore:27]: Where does .ficfg files come from? I don't recognize the file type.

File: .gitignore [hunk 2]
```diff
@@ -25,3 +25,4 @@ lib
 Win32
 Win64
 *.ficfg
+*.~*~
```
Comment (andersmelander) [inline .gitignore:28]: I think this is something you should add to your local ignore file.

**Comment by mWaltari** [comment]
True,

It is FixInsight config file. I'll try to remove that... Dunno hot to edit
these pull requests tough...

-Tee-

**Comment by andersmelander** [comment]
> Dunno hot to edit these pull requests tough...

You just did.

**Comment by andersmelander** [comment]
Normally I'd wait until someone else also approves the PR before I merge but from the state of the other PRs it doesn't look like this will happen.

**Pull request merged**